cylc/flow/task_pool.py

```diff
@@ -1493,11 +1487,22 @@ def can_spawn(self, name: str, point: 'PointBase') -> bool:
             LOG.debug(
                 'Not spawning %s/%s: before initial cycle point', point, name)
             return False
-        elif self.config.final_point and point > self.config.final_point:
+
+        if self.config.final_point and point > self.config.final_point:
             # Only happens on manual trigger beyond FCP
-            LOG.debug(
+            LOG.warning(
```

Comment:
Leftover debugging?

Or is this intended feedback for CLI use?

(just making sure this doesn't get logged for internal use of the method)

Reply:
(leftover debugging - reverted).